test/functional/wallet_backwards_compatibility.py

```diff
@@ -265,7 +265,12 @@ def run_test(self):
             )
             load_res = node_master.loadwallet("u1_v16")
             # Make sure this wallet opens without warnings. See https://github.com/bitcoin/bitcoin/pull/19054
-            assert_equal(load_res['warning'], '')
+            if '"warnings"' in node_master.help("loadwallet"):
```

Comment:
Is this the best way to test for this? Seems like it could be broken by someone using a word in a future doc update. You can't get `bitcoind --version` or something for the back-compat test?